data/json/items/ranged/crossbows.json

```diff
@@ -262,7 +262,7 @@
     "symbol": "(",
     "color": "green",
     "name": { "str": "wood crossbow" },
-    "description": "A moderately powerful all-wood hunting crossbow.",
+    "description": "An all-wood hunting crossbow.  It is not powerful enough to deal with zombies, but it may still be used for hunting."
```

Comment:
```suggestion
    "description": "An all-wood hunting crossbow.  It is not powerful enough to deal with zombies, but it may still be used for hunting.",
```